app/javascript/components/widgets/numbers/falling-blocks.jsx

```diff
@@ -344,6 +366,7 @@ class Scene extends React.Component {
     // randomly pick a point from one wall edge to the other
     // so that they don't fall directly on top of each other.
     if (block) {
+      setBlocksAddedSinceLastMount(true);
```

Comment:
oh I see, we are setting it from in here also. 

It might be easier to clear the world state, on unmount, if we haven't added anything new since it was mounted; clearing existing world cache when we don't need it, forces it to rebuild the next time it gets mounted. Could help avoid leaking more state into local storage

Reply:
Has anything been added was surprisingly tricky:
* Dropped balls on mount (server counts > local cache counts)
* Dropped balls while mounted (subscription event from server)
* New balls to drop while unmounted (server counts > local counts again)

Reply:
Ah, I see, that does seem tricky. Is it worth persisting it with the rest of the `world` state then?

Reply:
Maybe. I can't remember the reasoning we arrived at to prefer localStorage over local graphql storage. Do you remember @malssid

Reply:
I don't completely remember why. I think I chose it just because it's something I've used in the past.It might make sense to just store it with the world state.